jdbc-adapter/virtualschema-jdbc-adapter/src/test/java/com/exasol/adapter/dialects/oracle/OracleSqlDialectIT.java

```diff
@@ -1,25 +1,31 @@
 package com.exasol.adapter.dialects.oracle;
 
-import static org.junit.Assert.*;
-import static org.junit.jupiter.api.Assertions.assertThrows;
+import com.exasol.adapter.dialects.AbstractIntegrationTest;
+import com.exasol.adapter.dialects.IntegrationTestConfigurationCondition;
+import org.junit.Assume;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
 
 import java.io.FileNotFoundException;
 import java.math.BigDecimal;
 import java.net.URI;
-import java.sql.*;
 import java.sql.Date;
-import java.util.*;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
 
-import org.junit.Assume;
-import org.junit.jupiter.api.*;
-import org.junit.jupiter.api.extension.ExtendWith;
-
-import com.exasol.adapter.dialects.AbstractIntegrationTest;
-import com.exasol.adapter.dialects.IntegrationTestConfigurationCondition;
+import static org.junit.Assert.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
 
 /**
  * Tested with Oracle 12
- *
+ * <p>
  * TODO Add tests for data types TODO Test Expanding of SELECT * if elements of select list require casting
```

Comment:
Remove TODOs. If relevant --> move to tickets.

Reply:
done